website/src/pages/faqs/faq.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Overview
+keywords: [hudi, writing, reading]
+---
+# Overview
```

Comment:
why there are a lot of faq md addition in the diff? shouldn't they be just renaming of files or sth?

Reply:
but wouldn't there be also removal diff of the faq md files? are they still kept somewhere? we should only keep 1 copy. 

Also, is it possible make faq under "Learn" ?